projects/python/perception/object_detection_2d/nanodet/eval_demo.py

```diff
@@ -21,14 +21,15 @@
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument("--data-root", help="Dataset root folder", type=str)
-    parser.add_argument("--model", help="Model that config file will be used", type=str)
+    parser.add_argument("--model", help="Model that config file will be used", type=str, default="m")
     parser.add_argument("--device", help="Device to use (cpu, cuda)", type=str, default="cuda", choices=["cuda", "cpu"])
 
     args = parser.parse_args()
+    data_root = "/home/manos/data/coco2017"
```

Comment:
this should be changed to either something included with the demo, or instructions should be given on how to download it